backend-api/app/routers/system.py

```
@@ -66,38 +64,29 @@ async def startup_info():
-    page load. Forcing a fresh probe is handled by the re-run encoder tests
-    endpoint, which calls ``invalidate_hw_info_cache`` after running.
-    """
-    logger.debug("/api/hardware called")
+    """Get available hardware acceleration info from worker."""
     try:
-        info = await get_hw_info_cached_async()
-    except Exception as e:
-        logger.warning("/api/hardware: cached lookup failed: %s", e)
-        info = {"type": "cpu", "available_encoders": {}}
+        info = get_hw_info_fresh(timeout=5) or get_hw_info_cached()
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)